.github/workflows/agent-e2e.yml

```
@@ -0,0 +1,94 @@
+            );
+            
+            const message = await client.messages.create({
+              model: "claude-3-5-sonnet-20241022",
+              max_tokens: 4096,
+              messages: [{
+                role: "user",
+                content: `You are a fresh AI agent who has NEVER used Starknet before.
+                
+Your ONLY source of information is this skill guide:
+
+${skill}
```

actionlint (1.7.10)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)